lib/contracts/genesisProtocol.ts

```diff
@@ -95,7 +87,7 @@ export class GenesisProtocol extends ExtendTruffleContract(SolidityContract) {
     const tx = await this.contract.stake(
       options.proposalId
       , options.vote
-      , amount
+      , amount,
```

Comment:
not sure why you are adding trailing commas everywhere, but especially weird when we put commas in front

Reply:
linter